lm_eval/__main__.py

```diff
@@ -295,15 +301,22 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
         )
 
     if args.fewshot_as_multiturn and args.apply_chat_template is False:
-        raise ValueError(
-            "When `fewshot_as_multiturn` is selected, `apply_chat_template` must be set (either to `True` or to the chosen template name)."
+        # no chat_template means definitely no multiturn
+        args.fewshot_as_multiturn = False
+        eval_logger.warning(
+            (
+                "When `fewshot_as_multiturn` is selected, `apply_chat_template` must be set (either to `True` or to the chosen template name)."
+                "Disabling `fewshot_as_multiturn`."
+            )
         )
 
     if (
```

Comment:
I think we should just remove this condition as `num_fewshot` can be set in the yaml.

Reply:
totally agree. seems that if i provide num_fewshot=1 in yaml only, this condition misses it
will remove this if at all

Reply:
removed if for checking num_fewshots for multiturn runs
for zero-shot runs multiturn is equivalent to simple apply_chat_template run, so nothing should be broken